tokens/src/core/components/CloseButton.json

```diff
@@ -2,12 +2,8 @@
   "typography": {
     "close-button": {
       "font": {
-        "size": {
-          "value": "calc({typography.font.size.base} * 1.5)", "type": "dimension", "source": "$close-font-size"
-        },
-        "weight": {
-          "value": "{typography.font.weight.bold}", "type": "fontWeight", "source": "$close-font-weight"
-        }
+        "size": { "source": "$close-font-size", "$value": "calc({typography.font.size.base} * 1.5)", "$type": "dimension" },
```

Comment:
See https://github.com/openedx/paragon/pull/3186/files#r1739426413 for details on where we landed regarding `calc.`

tl;dr; there's no great alternative right now to avoid `calc` in style-dictionary tokens.

Reply:
How about creating an issue to track a specific CSS `calc()`?
This way, we won’t lose track of it in the future.

Reply:
Thanks @brian-smith-tcril!